debian/control

```diff
@@ -0,0 +1,25 @@
+Source: pymavlink
+Section: python
+Priority: optional
+Maintainer: Andrii Beregovenko <jet@jet.kiev.ua>
+Build-Depends: debhelper (>= 10),
+  dh-python,
+  python3-all-dev (>= 3.3.3),
+  libxml2-dev, libxslt1-dev, zlib1g-dev,
+  python3-setuptools (>= 0.6.29),
+  python3-numpy,
+  python3-pytest,
+X-Python-Version: all
+Standards-Version: 4.5.1
+Homepage: https://github.com/ArduPilot/pymavlink
+
+Package: python3-pymavlink
+Architecture: any
+Multi-Arch: same
+Depends: ${python3:Depends}, ${shlibs:Depends}, ${misc:Depends}, python3-future, python3-lxml
+Provides: ${python3:Provides}
+Description: Pymavlink is a Python implementation of the MAVLink protocol
```

Comment:
It really is three *distinct* things.

The most commonly used mavlink bindings generator.  Which also generates the Python mavlink bindings.

A set of utilities that make working with the Python bindings easier, most notably an object to handle opening data streams.

The command-line utilities that you've mentioned.

I don't think you really need to fix this in your stuff here - it's no different to many other places....

Reply:
Oh well, I just have to specify something here, thus stating what I can at my best :D
I would appreciate a more accurate Summary and Longer description.